lib/mayaUsd/fileio/utils/readUtil.cpp

```diff
@@ -356,6 +356,12 @@ MObject UsdMayaReadUtil::FindOrCreateMayaAttr(
     } else if (type.IsA<int>()) {
         return _FindOrCreateMayaNumericAttr(
             attrName, attrNiceName, MFnNumericData::kInt, keyable, usedAsColor, depNode, modifier);
+    } else if (type.IsA<unsigned int>()) {
```

Comment:
These fixes are unrelated to the bug, but were causing warnings in the script editor log because the attribute type was not supported.